common/src/test/java/tachyon/util/io/PathUtilsTest.java

```diff
@@ -16,13 +16,43 @@
 package tachyon.util.io;
 
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import tachyon.Constants;
+import tachyon.thrift.InvalidPathException;
 
 public class PathUtilsTest {
+  @Rule
+  public final ExpectedException mException = ExpectedException.none();
+
+  @Test
+  public void getPathComponentsTestSuccess() {
+    try {
```

Comment:
Do we still need `try`-`catch` here?
It seems that the test fails anyway when `InvalidPathException` is thrown.